DataConnectors/GCP/Terraform/sentinel_resources_creation/GCPInitialAuthenticationSetup/GCPInitialAuthenticationSetup.tf

```diff
@@ -77,10 +85,14 @@ resource "google_service_account_iam_binding" "bind-workloadIdentityUser-role-to
   role    = "roles/iam.workloadIdentityUser"
 
   members = [
-    "principalSet://iam.googleapis.com/projects/${data.google_project.project.number}/locations/global/workloadIdentityPools/${google_iam_workload_identity_pool.sentinel-workload-identity-pool.workload_identity_pool_id}/*",
+    "principalSet://iam.googleapis.com/projects/${data.google_project.project.number}/locations/global/workloadIdentityPools/${local.workload_identity_pool_id}/*",
   ]
 }
 
+output "An_output_message"{
```

Comment:
I think it should be given a better name

Reply:
Do you have any suggestion? It hasn't changed in this PR

Reply:
Msg_copy_values_to_sentinel
Just use the Msg or any other prefix incase you will have more output messages.
Any way since not changes this PR, next time :)

Reply:
Customers see this variable name, so it was product's decision. I'll ask her what she thinks :)